parser/ast/ddl.go

```diff
@@ -2160,11 +2163,14 @@ func (n *ResourceGroupOption) Restore(ctx *format.RestoreCtx) error {
 			ctx.WritePlain("= ")
 			ctx.WriteString(n.StrValue)
 		case ResourceBurstableOpiton:
-			ctx.WriteKeyWord("BURSTABLE")
+			ctx.WriteKeyWord("BURSTABLE ")
+			ctx.WritePlain("= ")
+			ctx.WritePlain(strings.ToUpper(fmt.Sprintf("%v", n.BoolValue)))
 		case ResourceGroupRunaway:
+			ctx.WritePlain("QUERY_LIMIT")
+			ctx.WritePlain(" = ")
```

Comment:
let's keep it consistent with others
```suggestion
			ctx.WritePlain("QUERY_LIMIT ")
			ctx.WritePlain("= ")
```